bot/cogs/snakes.py

```diff
@@ -29,6 +34,25 @@ def __init__(self, bot: AutoShardedBot):
         :return: A dict containing information on a snake
         """
 
+        # Getting the summary
+
+        async with aiohttp.ClientSession() as session:
+            async with session.get('https://en.wikipedia.org/w/api.php?action=parse&format=json&page=' + name + '&section=1') as resp:
```

Comment:
This line is too long. Try defining your URL on a separate line, and passing that into `session.get`.